contrib/build-rpm.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ "$(which rpmbuild)" == "" ];
+then
+  echo "To build an rpm the tool rpmbuild needs to be installed first"
+  exit -1
+fi 
+
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -z $1 ];
```

Comment:
use [[ instead of [. I'm not a bash expert but the difference relates to things like word splitting and wildcard expansion.